service-deployment/bootstrap/prometheus/start.sh

```diff
@@ -24,7 +24,7 @@ pushd $(dirname "$0") > /dev/null
 chmod u+x node-label.sh
 
 ./node-label.sh
-
+kubectl create configmap prometheus-alert --from-file=prometheus-alert
```

Comment:
>prometheus-alert [](start = 54, length = 16)

is this a folder? or a yaml file?

Reply:
it will load folder's .rule file, the file contains user define rules